src/content/promise-polyfill/promise-polyfill.mdx

```diff
@@ -0,0 +1,49 @@
+import Editor from './../../components/editor';
+
+## Implement Promise polyfill
+
+[Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise)
+
+> The Promise object represents the eventual completion (or failure) of an asynchronous operation and its resulting value.
+
+### Question
+
+Create a class `Promise` with two constructor parameters: `resolve` (fired when the promise is resolved) and `reject` (fired when the promise is rejected).
```

Comment:
@nitish24p is this description ok? Do they need to implement the polyfill part (checking windows object to see if Promise is defined)? And do they need to make the promise fully compliant (implementing `finally` and the other promise static methods)?